packages/ember-glimmer/lib/helpers/inline-if.js

```diff
@@ -0,0 +1,38 @@
+/**
+@module ember
+@submodule ember-templates
+*/
+
+import { toBool as emberToBool } from './if-unless';
+import { assert } from 'ember-metal/debug';
+
+/**
+  The inline `if` helper conditionally renders a single property or string.
+  This helper acts like a ternary operator. If the first property is truthy,
+  the second argument will be displayed, if not, the third argument will be
+  displayed
+  ```handlebars
+  {{if useLongGreeting "Hello" "Hi"}} Alex
+  ```
+  You can use the `if` helper inside another helper as a subexpression.
+  ```handlebars
+  {{some-component height=(if isBig "100" "10")}}
+  ```
+  @method if
+  @for Ember.Templates.helpers
+  @public
+*/
+export default function inlineIf(args) {
+  assert(
+   'The inline form of the `if` and `unless` helpers expect two or ' +
+   'three arguments, e.g. `{{if trialExpired \'Expired\' expiryDate}}` ',
+   args.length === 2 || args.length === 3
+ );
+
+  if (emberToBool(args[0])) {
+    return args[1];
+  } else {
+    let falsyArgument = args[2];
+    return falsyArgument === undefined ? '' : falsyArgument;
```

Comment:
I suppose this is needed to pass the tests at the moment? Can you add a TODO here? I believe we should unconditional return `args[2]` here (even if it's undefined) and let a different part of the system handle converting `undefined`, `null`, etc to `""` (which we haven't implemented yet).

It's probably good to add a test for this case if we don't already have one (I think it would have to be `@htmlbars` for now), but perhaps that should be part of the general [checklist item](https://github.com/emberjs/ember.js/issues/12907) of "Add tests for rending false, undefined, null etc in content-test.js, similar to..."

```javascript
this.render(`{{if condition truthy}}`, { condition: true, truthy: null });
this.render(`{{if condition truthy}}`, { condition: true, truthy: undefined });
this.render(`{{if condition truthy}}`, { condition: true, truthy: false });
...etc
```